docs/changelog.md

```diff
@@ -152,6 +152,15 @@ If you however want to continue using the `DefaultGraphManager`, then you can si
 setting again or change it to the TinkerPop `GraphManager` that has been the default before:
 `org.apache.tinkerpop.gremlin.server.util.DefaultGraphManager`.
 
+##### Removal of deprecated classes/methods/functionalities
+
+###### Classes
+
+* SchemaContainer class
```

Comment:
What about `EdgeLabelDefinition`, `PropertyKeyDefinition`, and `RelationTypeDefinition`?